test/integration/build/aws.tf

```diff
@@ -1411,6 +1413,33 @@ resource "aws_eks_cluster" "aws_eks_cluster" {
   }
 }
 
+resource "aws_subnet" "eks_subnet-3" {
```

Comment:
this should be renamed to `rds_subnet` or something more applicable

Reply:
Updated in recent commit